lte/gateway/c/core/oai/tasks/sgw/pgw_handlers.cpp

```diff
@@ -689,28 +689,35 @@ static void delete_temporary_dedicated_bearer_context(
                  spgw_context_p->sgw_eps_bearer_context().imsi64(),
                  "Delete temporary bearer context for lbi :%u \n", lbi);
 
-  int num_of_bearers_deleted = 0;
-  int num_of_pending_procedures = sgw_context_p->pending_procedures_size();
-  for (int proc_index = 0;
+  uint8_t num_of_bearers_deleted = 0;
+  uint8_t num_of_pending_procedures = sgw_context_p->pending_procedures_size();
+  for (uint8_t proc_index = 0;
        proc_index < sgw_context_p->pending_procedures_size(); proc_index++) {
     pgw_ni_cbr_proc = sgw_context_p->mutable_pending_procedures(proc_index);
+    if (!pgw_ni_cbr_proc) {
+      OAILOG_ERROR_UE(
+          LOG_SPGW_APP, spgw_context_p->sgw_eps_bearer_context().imsi64(),
+          "Pending procedure within sgw_context is null for proc_index:%u",
```

Comment:
Can you log bearer id here?

Reply:
In pending procedure list bearer id will not be allocated so logging s1u_teid